packages/create-sitecore-jss/src/templates/nextjs/src/byoc-imports/index.client.ts

```diff
@@ -0,0 +1,19 @@
+import * as FEAAS from '@sitecore-feaas/clientside/react';
+
+/**
+ * Below are built-in JSS imports. These should be available in Pages, if a license required for specific components is present
+ * Any built-in import can be removed if not used.
+ */
+
+// SitecoreForm component which enables new Sitecore Forms for XMCloud functionality. Requires a license to be available in Pages.
+// Sitecore Forms for Sitecore XP are still available separately via @sitecore-jss-forms package
+import '@sitecore/components/form';
+
+/**
+ * End of built-in JSS imports
+ * You can import your own client component below, for example
+ * import './MyClientComponent';
+ */
+
+// Client components bundle exporting an empty ExternalComponent enables proper BYOC registration and prevents nextjs from optimizing components away
+export default (props: unknown) => FEAAS.ExternalComponent(props);
```

Comment:
how about we move this on top instead? SO that all boilerplate is on top, instead of in the middle?

The commend i'd add:

`An important boilerplate component that prevents BYOC components from being optimized away. It is important to keep this export in this file`

Reply:
I'm not sure the imports would be correctly invoked in that case - better leave the export at the bottom.